requirements/doc.txt

```diff
@@ -18,7 +18,7 @@ attrs==21.4.0
     #   pytest
 babel==2.9.1
     # via sphinx
-bleach==4.1.0
+bleach==5.0.0
```

Comment:
Dropped support for `Python 3.6`. See the [CHANGELOG](https://github.com/mozilla/bleach/blob/main/CHANGES) for details.